ciphers/mixed_keyword_cypher.py

```diff
@@ -19,50 +21,49 @@ def mixed_keyword(key: str = "college", pt: str = "UNIVERSITY") -> str:
      'Y': 'T', 'Z': 'Y'}
     'XKJGUFMJST'
     """
-    key = key.upper()
-    pt = pt.upper()
-    temp = []
-    for i in key:
-        if i not in temp:
-            temp.append(i)
-    len_temp = len(temp)
-    # print(temp)
-    alpha = []
-    modalpha = []
-    for j in range(65, 91):
-        t = chr(j)
-        alpha.append(t)
-        if t not in temp:
-            temp.append(t)
-    # print(temp)
-    r = int(26 / 4)
-    # print(r)
+    keyword = keyword.upper()
+    plaintext = plaintext.upper()
+
+    unique_chars = []
+    for char in keyword:
+        if char not in unique_chars:
+            unique_chars.append(char)
```

Comment:
Please make this a `set`.

Reply:
It cannot be done with sets due to the nature of the algorithm. The order of those unique characters matters.

Reply:
I believe it cannot be done with sets because the order of these characters in the keyword matters - it determines how we will map plaintext characters to the cyphertext.

Reply:
> I believe it cannot be done with sets because the order of these characters in the keyword matters - it determines how we will map plaintext characters to the cyphertext.

Then I think it would help if this was mentioned in the comments, so that future maintainers know that the char ordeer matters